test/functional/page_objects/common_page.js

```
@@ -37,6 +37,39 @@ export function CommonPageProvider({ getService, getPageObjects }) {
   const defaultFindTimeout = config.get('timeouts.find');
 
   class CommonPage {
+
+    static async navigateToUrlAndHandleAlert(url, shouldAcceptAlert) {
+      log.debug('Navigate to: ' + url);
+      try {
+        await browser.get(url);
+      } catch(navigationError) {
+        log.debug('Error navigating to url');
+        let alert;
+        try {
+          alert = await browser.getAlert();
```

Now that getAlert swallows errors, the try/catch around it is not needed